action.yml

```diff
@@ -11,11 +11,14 @@ inputs:
     default: ".git" # Ignore the `.git` dirtectory by default
     required: false
   find:
-    description: "The string we want to replace"
+    description: "The regular expression (or fixed string) we want to replace"
     required: true
   replace:
     description: "The new string to replace with"
     required: true
+  fixed:
```

Comment:
Can we call this `regex` instead and flip the logic (still regex by default and set this to `false` to disable), I think `fixed` isn't very clear.

Reply:
Sure

Reply:
Could we also use glob matching in `with.include` and `with.exclude` when `regex: false` @jacobtomlinson?

Reply:
I would prefer those things to be controlled independently. 

To be honest I wonder if using regex for the file names is confusing anyway, perhaps that should just use glob (which would definitely require a bump to `v2`).

Reply:
Could we make this pull v2 then?

Reply:
v2 would ship with both features

Reply:
@jacobtomlinson ^ question about tagging

Reply:
Both features now implemented and documented!